README.md

```diff
@@ -1,2 +1,42 @@
 # app
 Contains a frontend web application that brings together all pieces.
+
+## How To Run It
+
+Make sure that you have NodeJS (>=14) and NPM installed.
+
+#### Clone
+
+Clone this repo to your local machine using
+```
+git clone https://github.com/remla23-team15/app.git
+```
+
+#### Run
+Move to  the application folder and run in your terminal:
+```
+# Add GitHub Personal Access Token to environment (linux)
+export NPM_TOKEN=the_token_goes_here
+
+# Add GitHub Personal Access Token to environment (windows)
+set NPM_TOKEN=the_token_goes_here
+```
+
+```
+# Docker build
+docker build -t ghcr.io/remla23-team15/app:VERSION --build-arg npm_token=${NPM_TOKEN} .
```

Comment:
Should we also add the command for windows?

docker build -t ghcr.io/remla23-team15/app:VERSION --build-arg npm_token=%NPM_TOKEN% .

Reply:
True! I forgot that one. I'll change it